plugins/hls-semantic-tokens-plugin/test/testdata/TDataFamily.hs

```diff
@@ -0,0 +1,11 @@
+{-# LANGUAGE TypeFamilies #-}
```

Comment:
Did you mean to also delete the old files?

Reply:
yes, I have delete them in the local, oddly I do have the old files in my local branch.

Reply:
I'll fix it in https://github.com/haskell/haskell-language-server/pull/3940